### ISSUE TITLE
UClient DTLS Permissions Issue

### ISSUE THREAD

**Issue by Lazarus404** [coturn/coturn#335]
Currently, when using UClient for TLS, TCP and UDP, a permissions packet is sent, but this doesn't happen for DTLS:

    turnutils_uclient -e 127.0.0.1 -L 127.0.0.1 -n 100 -p 5349 -v -S -l 2000 -k ./certs/server.key -i ./certs/server.crt -u user -w pass 127.0.0.1

Running the above, no permissions is ever set.  Is there a reason for this?

**Comment by misi** [comment]
Permission AFAIR is a general concept and not transport related. 
Unfortunately I am working on other stuff and not able to dig into it.
 
Many thanks for the report. 
Would it be possible for you to investigate it further?

Thanks!

**Comment by Lazarus404** [comment]
Certainly.  Permissions is not indeed transport related, which is why I find it odd.  The permission is always sent for the other three transports, just not DTLS.  I will investigate the C++ code to find the issue and will submit a patch once I find it.

**Comment by eakraly** [comment]
Hi @Lazarus404 ! Did you have a chance to investigate the issu?
